### DISCUSSION TITLE
Onboarding

### DISCUSSION THREAD

**Discussion by lukecater** [IBM/mac-ibm-notifications#81] [Q&A]
Hello, 
I currently use DEP notify when installing all necessary software prior to the user logging into the machine. Is is possible to do this with this?
I have played around with the onboarding and pop ups boxes which work great but only when a user is logged in. These will not show up on the login screen.

**Comment by SMartorelli** [comment]
Hi @lukecater, thanks for this report, we'll investigate this behaviour and I'll open a BUG for it in case!
